lib/cpus/aarch64/cortex_a53.S

```diff
@@ -228,11 +228,13 @@ endfunc cortex_a53_reset_func
 func cortex_a53_core_pwr_dwn
 	mov	x18, x30
 
+#if !HW_ASSISTED_COHERENCY
```

Comment:
Just one comment: Use a TI specific build flag in this file say "TI_AM65X_WORKAROUND".